docs/tutorials/migrations/nvm.md

```diff
@@ -40,17 +41,19 @@ Showing 10 of 67 results. Use `flox search nodejs --all` to see the full list.
 Use 'flox show <package>' to see available versions
 ```
 
-There is _a lot_ of software in the Flox Catalog, so you should have no trouble finding the node version you need. For this project, perhaps you decide to install Node.js v20.18.1.
+There is _a lot_ of software in the Flox Catalog, so you should have no trouble finding the node version you need. We recommend that you install one of the `nodejs` packages with a numerical suffix (e.g., `nodejs_18` or `nodejs_22`), rather than `nodejs`, which is the latest stable version of node from the perspective of the Nix Packages maintainers. The `nodejs` package may not get a new version until all the other packages that depend on that specific Node.js version have been updated for compatibility, which could be a long while after an official Node.js release.
```

Comment:
This sentence makes it sound more like it’s the responsibility of the upstream packages to support newer runtimes. My understanding is that it is the nix packaging that has to catch up (and maintain lower version bounds explicitly if a package doesn’t support a new runtime)